elastic/integrations
# Bugfix: Fix formulas for computing tier capacity

## Proposed commit message

Fix formulas for computing tier capacity



## Checklist


## How to test this PR locally



Install version 1.15.1 of this integration, check existing Tier Capacity visualizations, see that they are incorrect due to dividing by unique count of node names instead of multiplying.
